rdrf/rdrf/auth/__init__.py

```diff
@@ -8,4 +8,4 @@ def is_user_privileged(user):
 def can_user_self_unlock(user):
     return (
         getattr(settings, 'ACCOUNT_SELF_UNLOCK_ENABLED', False) and not
-        is_user_privileged(user) and not user.prevent_self_unlock)
+        is_user_privileged(user) and not user.is_curator and not user.prevent_self_unlock)
```

Comment:
Won't the is_user_privileged function return True for curators? i.e. they're not patients etc

Reply:
So adding an explicit check for is_curator seems redundant